terraform/pagerduty/modules/service/technical/main.tf

```diff
@@ -330,3 +330,36 @@ resource "aws_secretsmanager_secret_version" "pagerduty_alert_manager_integratio
   secret_id     = aws_secretsmanager_secret.pagerduty_alert_manager_integration_key[0].id
   secret_string = pagerduty_service_integration.alert_manager[0].integration_key
 }
+
+##################################################
+# Pingdom Integration
+##################################################
+
+data "pagerduty_vendor" "pingdom" {
+  name = "Pingdom"
+}
+
+resource "pagerduty_service_integration" "pingdom" {
+  count = var.enable_pingdom_integration ? 1 : 0
+
+  name    = data.pagerduty_vendor.pingdom.name
+  service = pagerduty_service.this.id
+  vendor  = data.pagerduty_vendor.pingdom.id
```

Comment:
We had some trouble with this a while back and ended up hardcoding them, see others

You can run https://github.com/ministryofjustice/analytical-platform/blob/main/scripts/pagerduty/vendor.sh to get its ID